prisma/schema.prisma

```diff
@@ -11,23 +11,23 @@ datasource db {
 }
 
 model Product {
-  id        Int      @id @default(autoincrement())
+  id        String      @id @unique
```

Comment:
uuid is already supported through primsa. you don't need another package to make this.
check prisma docs
https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#uuid